concourse/steps/upload_container_images.mako

```diff
@@ -48,17 +49,17 @@ component_descriptor_path = os.path.join(
 
 component_descriptor = parse_component_descriptor()
 
-image_filter = image_reference_filter(
```

Comment:
just extend the image_reference_filter function to accept the two additional arguments. rename to image_filter, maybe. caveat: name must not clash with variable name, of course.